piccolo_api/crud/serializers.py

```diff
@@ -39,6 +39,7 @@ def pydantic_json_validator(cls, value):
 @lru_cache()
 def create_pydantic_model(
     table: t.Type[Table],
+    exclude_columns: t.Set[str] = set(),
```

Comment:
One thing to be aware of is this will have to be a tuple instead.

It's because the function is wrapped in `lru_cache` it won't allow any mutable types.

You might consider passing in Piccolo columns rather than strings too, so something like this:

```python
exclude_columns: t.Tuple[Column, ...] = tuple(),
```

Reply:
Amazing, I'm so used to working with strings. This is much cleaner